docs/source/tutorial_notebooks/pipline_datasets.ipynb

```diff
@@ -0,0 +1,305 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9337f001-5e7c-4141-a60c-5e99052aee3d",
+   "metadata": {},
+   "source": [
+    "<img src=\"../_static/DREGS_logo.png\" width=\"300\"/>\n",
+    "\n",
+    "# Working with pipeline datasets\n",
+    "\n",
+    "This tutorial focuses on how to register data into `DREGS` from a\n",
```

Comment:
Did you really mean to call this notebook "pipline_datasets.ipynb"?   I would change to "pipeline_datasets.ipynb".